src/ZoneServer/Buffs/Handlers/EnchantEarth_Buff.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Melia.Shared.Tos.Const;
+using Melia.Zone.Buffs.Base;
+using Melia.Zone.Scripting;
+using Melia.Zone.World.Actors.Characters;
+
+namespace Melia.Zone.Buffs.Handlers
+{
+	/// <summary>
+	/// Handle for the Enchant Earth Buff, which increases the target's block penetration
+	/// </summary>
+	[BuffHandler(BuffId.EnchantEarth_Buff)]
+	public class EnchantEarth_Buff : BuffHandler
+	{
+		private float blockPenetrationBonus = 0;
```

Comment:
You can't save the modifier on the buff handler class, because this class is used globally by every player who gets this buff. You need to save it on the character or the buff, as other buffs are already doing.